extensions/wikia/Email/EmailImageHelper.class.php

```diff
@@ -34,9 +55,9 @@ class ImageHelper {
 	public static function getIconInfo() {
 		$info = [];
 
-		foreach ( self::$icons as $name ) {
-			$fileInfo = self::getFileInfo( $name . '.gif' );
-			$info[$name] = $fileInfo;
+		foreach ( self::$icons as $key => $icon ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)